kuma/wiki/tests/test_views.py

```diff
@@ -3390,7 +3390,7 @@ class MindTouchRedirectTests(UserTestCase, WikiTestCase):
          'expected': '/zh-CN/docs/XHTML'},
         {'title': 'JavaScript', 'mt_locale': 'zh_cn', 'kuma_locale': 'zh-CN',
          'expected': '/zh-CN/docs/JavaScript'},
-        {'title': 'XHTML6', 'mt_locale': 'zh_tw', 'kuma_locale': 'zh-CN',
+        {'title': 'XHTML6', 'mt_locale': 'zh_tw', 'kuma_locale': 'zh-TW',
```

Comment:
It was very strange that this worked! However, I see that the test was just looking at the first redirect, and matching it again the ``expected`` value ``/zh-TW/docs/XHTML6``. Only took me 15 minutes to figure that out 🙁